src/ProcessMaker/Nayra/Contracts/Storage/BpmnDocumentInterface.php

```diff
@@ -9,6 +9,9 @@
  * BPMN DOM file interface
  *
  * @package \ProcessMaker\Nayra\Contracts\Storage
+ * 
+ * @method DOMNodeList getElementsByTagName(string $name)
```

Comment:
The methods referenced in the annotation aren't present in the file

Reply:
That's because DomDocument does not have an Interface defined, and this annotations are used to help the developer to access common methods from that interace.